core/src/main/java/com/statful/client/core/StatfulClientImpl.java

```diff
@@ -93,6 +93,17 @@ public final SenderFacade gauge(final String metricName, final Integer value) {
     }
 
     @Override
+    public final SenderFacade metric(final String metricName, final Integer value) {
```

Comment:
Since the `metric` or `put` is just a construct and the `send()` is where the metrics will be effectively put into to the buffer. If we prefer consistency, I think we should use `put` then, even if semantically isn't perfect. IMO, changing all other clients because of this, doesn't make sense. 😉 I'll refactor it to `put`.